# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1450370562 18000
#      Thu Dec 17 11:42:42 2015 -0500
# Node ID 1da6c81e6486d08213999491eb67e12f919fee1c
# Parent  5df1d099e56db29af7a3b73b486ee4dafa6bc3f4
style fix for function declarations

* sleep.m, usleep.m, isocolors.m, isonormals.m:
Write "function foo (...)" instead of "function [] = foo (...)".

diff --git a/scripts/deprecated/sleep.m b/scripts/deprecated/sleep.m
--- a/scripts/deprecated/sleep.m
+++ b/scripts/deprecated/sleep.m
@@ -22,17 +22,17 @@
 ## @code{sleep} is deprecated and will be removed in Octave version 4.6.
 ## Use @code{pause} instead.
 ##
 ## Suspend the execution of the program for the given number of seconds.
 ##
 ## @seealso{pause}
 ## @end deftypefn
 
-function [] = sleep (seconds)
+function sleep (seconds)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "sleep is obsolete and will be removed from a future version of Octave, please use pause instead");
   endif
 
diff --git a/scripts/deprecated/usleep.m b/scripts/deprecated/usleep.m
--- a/scripts/deprecated/usleep.m
+++ b/scripts/deprecated/usleep.m
@@ -23,17 +23,17 @@
 ## Use @code{pause} instead.
 ##
 ## Suspend the execution of the program for the given number of
 ## microseconds (1e-6 seconds).
 ##
 ## @seealso{pause}
 ## @end deftypefn
 
-function [] = usleep (microseconds)
+function usleep (microseconds)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "usleep is obsolete and will be removed from a future version of Octave, please use pause instead");
   endif
 
diff --git a/scripts/plot/draw/isocolors.m b/scripts/plot/draw/isocolors.m
--- a/scripts/plot/draw/isocolors.m
+++ b/scripts/plot/draw/isocolors.m
@@ -46,17 +46,17 @@
 ## argument to all variations of function calls instead of the vertices
 ## data @var{v}.  Finally, if no output argument is given then directly
 ## change the colors of a patch that is given by the patch handle
 ## @var{p}.
 ##
 ## For example:
 ##
 ## @example
-## function [] = isofinish (p)
+## function isofinish (p)
 ##   set (gca, "PlotBoxAspectRatioMode", "manual", ...
 ##             "PlotBoxAspectRatio", [1 1 1]);
 ##   set (p, "FaceColor", "interp");
 ##   ## set (p, "FaceLighting", "flat");
 ##   ## light ("Position", [1 1 5]);  # Available with JHandles
 ## endfunction
 ##
 ## N = 15;    # Increase number of vertices in each direction
diff --git a/scripts/plot/draw/isonormals.m b/scripts/plot/draw/isonormals.m
--- a/scripts/plot/draw/isonormals.m
+++ b/scripts/plot/draw/isonormals.m
@@ -45,17 +45,17 @@
 ##
 ## If no output argument is given then directly redraw the patch that is
 ## given by the patch handle @var{p}.
 ##
 ## For example:
 ## @c Set example in small font to prevent overfull line
 ##
 ## @smallexample
-## function [] = isofinish (p)
+## function isofinish (p)
 ##   set (gca, "PlotBoxAspectRatioMode", "manual", ...
 ##             "PlotBoxAspectRatio", [1 1 1]);
 ##   set (p, "VertexNormals", -get (p,"VertexNormals")); # Revert normals
 ##   set (p, "FaceColor", "interp");
 ##   ## set (p, "FaceLighting", "phong");
 ##   ## light ("Position", [1 1 5]); # Available with JHandles
 ## endfunction
 ##
